.changes/next-release/feature-AWSSDKforJavav2-14a753c.json

```diff
@@ -0,0 +1,5 @@
+{
+    "category": "AWS SDK for Java v2", 
```

Comment:
nit, category should be `Amazon S3` maybe?

Reply:
Fuxed!